From ecd7e4f42c33943be0f9e355e1f1434723f84cd2 Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Wed, 11 Jun 2008 17:52:33 +0800
Subject: [PATCH 007/280] ENGR00079197 incorrect OVL_INTERNAL_BUF size causes
 some conformance test fail

BUGFIX
incorrect OVL_INTERNAL_BUF size causes some conformance test failure
OVL_USE for Marley should be 160*MB width (byte), not 80*MB width (byte)

Signed-off-by: Robby Cai <r63905@freescale.com>
---
 vpu/vpu_lib.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vpu/vpu_lib.c b/vpu/vpu_lib.c
index f7bae0c..70ecc2e 100644
--- a/vpu/vpu_lib.c
+++ b/vpu/vpu_lib.c
@@ -195,7 +195,7 @@ RetCode vpu_Init(PhysicalAddress workBuf)
 
 /* the following size is enough for worst case */
 #define	DBK_USE_SIZE	(0x6400 * (USE_DBK_INTERNAL_BUF != 0))
-#define	OVL_USE_SIZE	(0x1000 * (USE_OVL_INTERNAL_BUF != 0))
+#define	OVL_USE_SIZE	(0x2000 * (USE_OVL_INTERNAL_BUF != 0))
 #define IP_USE_SIZE	(0x1900 * (USE_IP_INTERNAL_BUF != 0))
 #define BIT_USE_SIZE	(0x1900 * (USE_BIT_INTERNAL_BUF != 0))
 
-- 
1.8.0

